build.xml

```diff
@@ -34,8 +34,8 @@
         <phingcall target="composer-check" />
     </target>
 
-    <target name="standards" depends="phplint,ecs,phpstan,twig-lint,yaml-lint,eslint-check,standards-packages" description="Checks coding standards in the project-base, packages and microservices."/>
-    <target name="standards-diff" depends="phplint-diff,ecs-diff,phpstan,twig-lint-diff,yaml-lint,eslint-check-diff,standards-packages" description="Checks coding standards on changed files in the project-base, packages and microservices."/>
+    <target name="standards" depends="phplint,ecs,phpstan,twig-lint,yaml-lint,eslint-check,standards-packages,check-correct-yaml-indents" description="Checks coding standards in the project-base, packages and microservices."/>
```

Comment:
shouldnt this be part of coding-standards package? @vitek-rostislav @TomasVotruba